### ISSUE TITLE
No Database

### ISSUE THREAD

**Issue by Curtis-Jiang-2020** [hankinsoft/SQLPro#494]
App shows no database when there are actually multiple postgres databases (few of them are empty)

![image](https://user-images.githubusercontent.com/68557591/92562707-65552c00-f244-11ea-8d10-533eb034c985.png)

Tried clear cache and restart, not working.

BTW postgres connection works in PyCharm, and I can see all schemas there.

**Comment by hankinsoft** [comment]
Hi.
Do you have a database selected if you edit the connection?
What do you receive for results if you run the following query:

`SELECT * FROM pg_database WHERE datistemplate = false;`

**Comment by Curtis-Jiang-2020** [comment]
Yes I have. If I edit that database name to something else, it will prompt "no such database xxx"

This SQL returns more than one database name that I can also see under connection edit's auto-completion, but if I use that name the connection fails with "no such database xxx". Sorry for unable to share screenshots.

BTW under PyCharm I can unfold these databases and see the schemas inside.

**Comment by hankinsoft** [comment]
Hmmm, what happens if you enter no database name at all when editing the connection? Does it then list all of them when you connect?

**Comment by Curtis-Jiang-2020** [comment]
The strange thing is it shows up the database name (same as pycharm), but I cannot use them as database name, otherwise it will not connect:
![image](https://user-images.githubusercontent.com/68557591/93143163-627b9080-f6b5-11ea-891d-5179850e1910.png)
This database name is differ from the one I can use to connect, and it is shown in PyCharm connection manager's schema section, not database connection section.
![image](https://user-images.githubusercontent.com/68557591/93143268-948cf280-f6b5-11ea-98e7-bd385e1430e2.png)
e.g. The three database above (in schema section) is shown in SQLPro's auto completion when editing connections, but non of these are the database name I'm using in successful postgres connection (the connection URL)

**Comment by hankinsoft** [comment]
Sorry, I'm still not quite understanding.

If you enter no database name in the SQLPro connection? Does it show them all in the database tree? Or does it still list no databases? Or that's when you see the 'error no such database'?

**Comment by Curtis-Jiang-2020** [comment]
It will show 'error no such database:xxxx'.

**Comment by hankinsoft** [comment]
I've made some changes to how postgres lists databases in the recent build. Could you give it a try and let me know if you are still having this issue?
